src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -515,6 +587,30 @@ public virtual RedirectToActionResult RedirectToAction(string actionName, string
         }
 
         /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status307TemporaryRedirect"/>) to the specified action with 
+        /// <see cref="RedirectToActionResult.Permanent"/> set to false and <see cref="RedirectToActionResult.PreserveMethod"/> 
+        /// set to true, using the specified <paramref name="actionName"/>, <paramref name="controllerName"/>, 
+        /// <paramref name="routeValues"/>, and <paramref name="fragment"/>.
+        /// </summary>
+        /// <param name="actionName">The name of the action.</param>
+        /// <param name="controllerName">The name of the controller.</param>
+        /// <param name="routeValues">The parameters for a route.</param>
```

Comment:
BTW is this the comment we usually use for this parameter, such as in other places in MVC? The phrasing here seems odd.